src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -403,6 +386,53 @@ class SingleActiveFIMap extends React.Component<
     );
   }
 
+  private async fetchData() {
+    const {
+      fetchGoalsActionCreator,
+      fetchJurisdictionsActionCreator,
+      fetchPlansActionCreator,
+      fetchStructuresActionCreator,
+      fetchTasksActionCreator,
+      plan,
+      supersetService,
+    } = this.props;
+
+    if (plan && plan.plan_id) {
+      /** define superset filter params for jurisdictions */
+      const jurisdictionsParams = superset.getFormData(SUPERSET_MAX_RECORDS, [
+        { comparator: plan.jurisdiction_id, operator: '==', subject: 'jurisdiction_id' },
```

Comment:
Is `jurisdiction_id` a magic string in this case? I think so, please move the `subject` strings into the `/src/constants.tsx` file 🙏